uber/models.py

```diff
@@ -1339,7 +1339,7 @@ def is_unassigned(self):
 
     @property
     def is_dealer(self):
-        return self.ribbon == c.DEALER_RIBBON or self.badge_type == c.PSEUDO_DEALER_BADGE
+        return self.ribbon == c.DEALER_RIBBON or self.badge_type == c.PSEUDO_DEALER_BADGE or self.group and self.group.is_dealer and self.paid == c.PAID_BY_GROUP
```

Comment:
I think this may require parenthesis around the group clause of the condition:
```python
return self.ribbon == c.DEALER_RIBBON or self.badge_type == c.PSEUDO_DEALER_BADGE or (self.group and self.group.is_dealer and self.paid == c.PAID_BY_GROUP)
```

Also, I kinda want to start writing horizontally shorter code, so maybe something like this?

```python
return (self.ribbon == c.DEALER_RIBBON or 
    self.badge_type == c.PSEUDO_DEALER_BADGE or 
    (self.group and self.group.is_dealer and self.paid == c.PAID_BY_GROUP))
```

Finally, if we're updating what the `is_dealer` property means, we should probably update the tests for `is_dealer`? Maybe test every combination of each part of that conditional as True/False?